script/verify.py

```diff
@@ -139,6 +139,8 @@ async def authorize_user(self, ctx, logs, name, entry_number):
             auth_url = google_auth_service.auth_url()
         except Exception as err:
             await logs.print(err)
+            return False
```

Comment:
if we do not have an auth_url or google auth service is not initialized, there is no point in continuing the function.

Reply:
ok